Origin/chapters1and2 (#66)
SanliFaez/OS4Physicists: Exercises_2023.md

# Open Science for Physicists
*Overview of assignments and tutorial activities*

Most of the exercises in this course are done in teams, which are either chosen by the coordinators or self-organize. To evaluate individual contributions, teams should also post a CREDIT decleration when posting each assignment.

## Week 1: why open science, how did we get here?  
*keywords:* Five schools of Open Science, Current state of the conversation (NL, UU, Physics), Amateurism, Laboratory life, The reproducibility crisis, Fraud, Access and Emancipation, Community-Discipline-Employer, International context, Transdsiciplinary and interdisciplinary science, Risks and rewards of Open Science, Oposition and counter arguments

+ Learning basic git: students are guided to join the course repository as collaborators and to make their first pull request
+ Reflection: Students collet the new concepts from the lecture and discuss them in the break-out sessions. In the plenary, they are asked to reflect on what they collective know about these concept and if that is enough for their career decisions.

> + All the students collective write the code of conduct of the course, first in break-out sessions, and then all together.

Reviewer: ```suggestion
+ All the students collectively write the code of conduct of the course, first in break-out sessions, and then all together.
```

## Week 2: setting up a collaborative environment
*keywords:* Version control, Licensing and recieving credit, Readme files and templates, Best practices for community management, Active and passive versioning, Project management and Getting Things Done (GTD), Unit testing (defensive programming), Merging forked repositories 

> + In two big teams, studens have to fork a python project that needs troubleshooting, then fix it in different stages taking charge of coordination and task distribution.

Reviewer: These would be _really_ big teams, upwards of 40 students. We could decide to make it three or four teams?

+ Every student forks the final project from their team, runs it in their PC, and they submit the individual answer under the corresponding issue on the course repo.

> + To prepare for week 3, teams will be assign a big scientific collaboration to investigate and report about in week 3.

Reviewer: ```suggestion
+ To prepare for week 3, teams will be assigned one big scientific collaboration (e.g. ATLAS@CERN, IPCC, Ligo/Virgo...) to investigate and report about in week 3.
```

## Week 3: big science, scale vs precision vs control
*keywords:* Big-science collaborations, Citizen science, Credit and recognition, Policy and politics of science, Governance structures, Independence and thriving, Relativism of big, Climate impacts and transformation of the research ecosystem

> + Students investigate the structure, history, function, and working conditions of these big collaborations and present for the class. Presentations will recieve peer-feedback.

Reviewer: ```suggestion
+ Students investigate the structure, history, function, and working conditions of these big collaborations and present for the class. Presentations will recieve peer-feedback.
Possible projects include (names in italics not yet contacted)
```

> 	+ IPCC, Roderick van der Wal, Aimee Slangen,
> 	+ ATLAS, Alessandro Grelli, 
> 	+ LIGO/VIRGO, Chris van den Broeke
> 	+ NEEM, ?? 
> 	+ Spectroscopy, Frans Snik
> 	+ Astroplant https://www.astroplant.io/, (?)
> 	+ Pangeo, Ryan Abernati
> 	+ SKA, 
> 	+ Chinese space agency,
> 	+ ECMWF (KNMI), 
> 	+ Seti at home,
> 	+ Large scale environment sensing, Rolf Hut

Reviewer: ```suggestion
	+ IPCC, _Roderik van de Wal_, _Aimée Slangen_
	+ ATLAS, _Alessandro Grelli_
	+ LIGO/VIRGO, _Chris van den Broeke_
	+ NEEM, _Carina van der Veen_
	+ Spectroscopy, _Frans Snik_
	+ Astroplant https://www.astroplant.io/, (?)
	+ Pangeo, _Ryan Abernathey_
	+ SKA, 
	+ Chinese space agency, _Marc Klein Wolt_
	+ ECMWF (KNMI), _Rianne Giesen_
	+ Seti @ home, ??
	+ Large scale environment sensing, _Rolf Hut_
```

> + How is open science implement in the structure of the big collaborations, and does it help?

Reviewer: ```suggestion
+ How is open science implemented in the structure of the big collaborations, and does it help?
```

+ Student teams start with their measurement project, define the goal and the processes, and create the documentation folder by following the lecture tips.

## Week 5: open design and open hardware; prototyping
*keywords:* Scientific design environments, Open protocols, Microcontrollers, Human-computer interface, Automation and bots, Remote labs, Materials-resources-constraints, Defense and national security, Maintenance, Prototyping

+ Each team presents their project to the class
+ Teams have to reproduce each-others results and write a review on 

## Week 6: public engagement: story-telling and presentation; activism and trust in scientists
*keywords:* How storytelling works, Science communication and journalism, Activism and trust in Science, Bounds of expertise, Academic social contract, Metaphores, Communicating uncertainty, Narratives and framing, Difficult to reach audience, Process and outcome

> + lay person summary of a previous or future project

Reviewer: ```suggestion
+ each student writes a lay person summary of one of their previous or future projects
```

> + Identify jargon and discuss how to avoid it in the science communication (inspiration from upgoer 5)

Reviewer: ```suggestion
+ In pairs, the students identify the jargon in their layperson summaries, and discuss how to avoid it in the science communication (inspiration from upgoer 5)
```

> + assignments will be test using language level tools

Reviewer: ```suggestion
+ assignments will be tested using language level tools (e.g. https://github.com/textstat/textstat, tbd)
```

## Week 7: from science to business: tech transfer (including open strategies), intelectual property and the academic social contract, scipreneurship
*keywords:* Technology transfer, Open strategies and open science business models, Scipreneurship; starting a business, Creating impact, Forking vs contributing, Moral ambition, IP-Licensing-Conflict of interest

> + (individually) make an open (business) canvas about the potential users of the master thesis topic or a project?

Reviewer: ```suggestion
+ Students individually make an open (business) canvas about the potential users of the master thesis topic or a project
```
